DESCRIPTION

```diff
@@ -21,10 +21,7 @@ Imports:
 Suggests:
     highr,
     Cairo,
-    testthat,
-    data.table,
-    dplyr,
-    htmlwidgets
+    testthat
 Enhances: data.table, dplyr, htmlwidgets
```

Comment:
maybe in a later version, we should switch to https://github.com/hadley/tibble in the tests...